be/src/exprs/expr.h

```diff
@@ -182,6 +182,10 @@ class Expr {
                                                         Expr* parent, int* node_idx, Expr** root_expr,
                                                         ExprContext** ctx, RuntimeState* state);
 
+    [[nodiscard]] static Status create_tree_from_thrift_with_jit(ObjectPool* pool, const std::vector<TExprNode>& nodes,
```

Comment:
nodiscard is not necessary because we have add this keyword before the definition of Status class

Reply:
ok